demos/cypress-shared/tsconfig.json

```diff
@@ -14,6 +14,9 @@
         },
         {
             "path": "../../packages/util"
+        },
+        {
+            "path": "../../packages/procaptcha-frictionless"
```

Comment:
Is this required? You have no imports from @procaptcha-frictionless in the cypress code.

Reply:
Sorry, I didn't removed it after test
I will remove these